### ISSUE TITLE
Tab Session Manager should show tree structure

### ISSUE THREAD

**Issue by mzso** [sienori/Tab-Session-Manager#1098]
**Is your feature request related to a problem? Please describe.**
This wold be both for usability and trouble shooting. Since I use TST and TSM does save the structure, it would be useful to actually see it. One might only want a tab or two and not want to open a whole session for that. And showing the tree structure would aid in that a lot.
Also since apparently TST has a bug (#846) with saving and restoring trees. It would be really helpful to troubleshoot. To verify whether the tree structure is even saved properly, or whether it's only the restoration that fails.

**Describe the solution you'd like**
A tiny amount of indentation for branches would do wonders. If branches were also collapsible, it would be even better. There's plenty of space if TSM is opened in a tab. For the doorhanger in my opinion there's too much space assigned to the session names anyway. They're usally just sort descriptive titles one adds. And tooltips can always show the whole on hover if someone uses a long one.

**Describe alternatives you've considered**
No real alternatives for this. Either you can see the tree structure or you can't.

**Additional context**

**Comment by deryb** [comment]
I agree, along that there should be the usual tree-lst features:  expand + collapse, close (x) & remove child tabs (of a whole group).  It only makes sense, since you might want to quick clean-up a session before restoring it.
